src/java/cheshire/prettyprint/CustomPrettyPrinter.java

```diff
@@ -0,0 +1,87 @@
+package cheshire.prettyprint;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.util.DefaultIndenter;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+
+import java.io.IOException;
+
+class DynamicIndenter extends DefaultIndenter {
+    public DynamicIndenter(int indentation) {
+        super(new String(new char[indentation]).replace("\0", " "), "\n");
```

Comment:
I can. I am not a Java guy. I know how to repeat a string properly in Clojure, JS and Python. I search for "java string repeat" and [this](http://stackoverflow.com/questions/1235179/simple-way-to-repeat-a-string-in-java/4903603#4903603) is the most upvoted SO answer.